.hound.yml

```diff
@@ -0,0 +1,3 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
```

Comment:
please add this `ignore_file: .eslintignore` so hound does look beyond eslint

Reply:
I will add that to the file now.